src/Bridge/Symfony/Bundle/EventListener/SwaggerUiListener.php

```diff
@@ -34,4 +34,10 @@ public function onKernelRequest(GetResponseEvent $event)
 
         $request->attributes->set('_controller', 'api_platform.swagger.action.ui');
     }
+
+    public function addEndpointTypeToRequest(GetResponseEvent $event)
```

Comment:
Is it necessary? The flag will always be set for every request even if it's not caught by Swagger UI isn't it?

Reply:
Oh, I need to set the same check as `onKernelRequest` function.

I need to separate the two listener because the AddFormatListener come between priority 0 and 7